components/admin_console/admin_definition.jsx

```diff
@@ -199,10 +199,7 @@ const AdminDefinition = {
                 'Mattermost Enterprise Edition. Unlock enterprise features in this software through the purchase of a subscription from ',
                 'This software is offered under a commercial license.\n\nSee ENTERPRISE-EDITION-LICENSE.txt in your root install directory for details. See NOTICE.txt for information about open source software used in this system.',
             ],
-            isHidden: it.any(
-                it.not(it.enterpriseReady),
-                it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
-            ),
+            isHidden: it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
```

Comment:
There is actually a `isHidden` check above on line 187 that says 
`isHidden: it.not(it.userHasReadPermissionOnResource('about')),` 

so you could remove that line and combine it with this check here to be something like
```
isHidden: it.any(
  it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
  it.not(it.userHasReadPermissionOnResource('about')),
),
```